src/main/java/step2/domain/LottoShuffleManager.java

```diff
@@ -0,0 +1,14 @@
+package step2.domain;
+
+import java.util.Collections;
+import java.util.List;
+
+public class LottoShuffleManager {
+	public static List<LottoNumber> shuffleSixNumbers() {
+		List<LottoNumber> numberList = LottoNumber.getLottoBallNumbers();
+		Collections.shuffle(numberList);
+		List<LottoNumber> lottoNumbers = numberList.subList(0, 6);
```

Comment:
이미 상수로 잘 정의해주신 값을 사용하는게 더 좋을 것 같습니다 🙇 
```suggestion
		List<LottoNumber> lottoNumbers = numberList.subList(0, LottoConstant.LOTTO_COUNT);
```